.github/workflows/api-gov.yml

```diff
@@ -13,11 +26,26 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Set version
+
+      - name: Set version to write baseline
+        if: "${{ inputs.update_baseline }}"
         run: |
           VER=$(cat ./packages/sn-filter-pane/package.json | jq '.version' | tr -d '"')
           echo "VER=${VER}" >> $GITHUB_ENV
 
+      - name: Set version to verify
+        if: "${{ !inputs.update_baseline }}"
+        run: |
+          yarn install --immutable --immutable-cache --check-cache
+          VER=`node ./scripts/get-pre-next-version.js`
```

Comment:
I can not see any get-pre-next-version.js file (forgot to add?)

Reply:
Yes, forgot to add